ticdc/ticdc-compatibility.md

```diff
@@ -7,10 +7,25 @@ summary: 了解 TiCDC 兼容性相关限制和问题处理。
 
 本文介绍了与 TiCDC 有关的一系列兼容性问题及其处理方案。
 
-<!--
-## 组件兼容性矩阵
+## TiCDC 和其他 TiDB 组件的兼容性
 
-TODO
+### TiCDC 和 Lightning 的兼容性
+
+Lightning 支持[逻辑导入模式](/tidb-lightning/tidb-lightning-logical-import-mode.md)和[物理导入模式](/tidb-lightning/tidb-lightning-physical-import-mode.md)。
+
+逻辑导入模式以向 TiDB 执行 SQL 语句的方式导入数据。这部分数据可以正常被 TiCDC 同步。操作步骤如下：
+
+1. 创建 changefeed
+2. 启动 Lightning，以逻辑模式模式导入数据
+
+物理导入模式以向 TiKV 插入 SST 文件的方式导入数据。当前阶段，TiCDC 和 Lightning 物理模式并不兼容，不同步由该模式导入的数据。
+
+如果下游集群是 TiDB，操作步骤如下：
+
+1. 分别向上下游 TiDB 集群用 Lightning 导入数据，保证上下游集群的数据一致性
+2. 创建 changefeed，同步上游集群后续通过 SQL 写入的数据到下游
+
+如果下游是非 TiDB 系统，需要先将 Lightning 的输入文件内容通过对应系统的离线导入工具导入到下游系统中，再创建 changefeed 以同步 SQL 写入的增量数据。
 
```

Comment:
Please remove the following lines (L30-L34)
https://github.com/pingcap/docs-cn/blob/b93b7b682f1b5605152689ff3c80a5dcc6e63b1c/ticdc/ticdc-compatibility.md?plain=1#L30-L34